opentech/apply/review/models.py

```diff
@@ -72,3 +74,25 @@ def __str__(self):
 
     def __repr__(self):
         return f'<{self.__class__.__name__}: {str(self.review)}>'
+
+
+@receiver(post_save, sender=Review)
+def update_submission_reviewers_list(sender, **kwargs):
+    review = kwargs.get('instance')
+
+    if not review.submission.reviewers.filter(id=review.author.id).exists():
+        review.submission.reviewers.add(review.author)
+        review.submission.save()
```

Comment:
You dont need to save after an m2m add. Unless there is something else you are trying to trigger?